rholang/src/main/scala/coop/rchain/rholang/interpreter/ContractCall.scala

```diff
@@ -9,6 +9,21 @@ import coop.rchain.rholang.interpreter.storage.implicits.matchListPar
 import coop.rchain.rspace.util.unpackCont
 import cats.implicits._
 
+/**
+  * This is a tool for unapplying the messages sent to the system contracts.
```

Comment:
Thanks for adding this. Normally, when I use `/** Enter`, ScalaDoc autogenerates `@param` tags.

Reply:
I had those but I removed them because I didn't know what to write except for
```  
  * @param space the rspace instance
  * @param dispatcher the dispatcher
```
But if you insist I'm adding them back

Reply:
That's ok. Don't worry about it!